core-v1/src/main/scala/dsl/standard.scala

```diff
@@ -4,11 +4,16 @@ package dsl
 
 import debug.HasSourceCodeInfo
 
-object standard extends BuildExprDsl with StandardRunDsl {
+case object standard extends UnwrappedBuildExprDsl with StandardRunDsl {
```

Comment:
TBH, I don't really like the name "standard" here... I don't think having every aspect of the computation captured in a mirror `ExprResult` tree is really "standard". It's more like "lossless" or maybe "mirrored" would be a better name... Either way, this is outside the scope of this PR. I would need to rename a lot of stuff all at once, so this would be a breaking pre-v1 change after I make all the other changes.